app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find params[:id]
```

Comment:
Throw an exception.  I guess better UX would be using find_by and then checking if we have a person object or nil `unless @quote.nil?` or something and redirecting to either the quote or a no quote page depending on if it's there or not.